### ISSUE TITLE
"There is already a provision request for app heroku-prisma"

### ISSUE THREAD

**Issue by marktani** [prisma/prisma-cloud-feedback#174]
![image](https://user-images.githubusercontent.com/1780597/42313030-07acf090-8042-11e8-841a-01d71b09869f.png)

Related: https://github.com/prismagraphql/prisma-console/issues/41

**Comment by tcbyrd** [comment]
@marktani 👋I did some testing with this last night and I imagine this is happening because the app `heroku-test` and `heroku-prisma` already exists in the global heroku namespace. I was able to get it working by adding my username in front of the app name (`tcbyrd-heroku-test`). You get a "Name is already taken" error when you try to create apps with names that have probably already been taken by  other people trying this out.

Response from the Heroku CLI where you get a similar error:

```shell
➜ prisma-test: heroku apps:create heroku-test
Creating ⬢ heroku-test... !
 ▸    Name is already taken
➜  prisma-test: heroku apps:create heroku-prisma
Creating ⬢ heroku-prisma... !
 ▸    Name is already taken
➜  prisma-test heroku apps:create tcbyrd-heroku-prisma
Creating ⬢ tcbyrd-heroku-prisma... done
```

It should probably attempt to create an app with a unique ID, or at least validate against Heroku on the previous screen to check if the app name is already taken.
